### DISCUSSION TITLE
multiple BSSID-addresses in nzyme.conf

### DISCUSSION THREAD

**Discussion by DMBrosig** [nzymedefense/nzyme#525] [Questions] [answered]
Hi, 
how can I configure multiple BSSID addresses for one WLAN. There is a example for fingerprints but not for addresses.

Thanks Dirk

**Comment by lennartkoopmann** [comment, marked as answer]
I've added a section for that to the documentation: https://www.nzyme.org/docs/installation-configuration/network-monitoring

Let me know if that solves the problem.
